kotlintest-runner/kotlintest-runner-junit5/src/main/kotlin/io/kotlintest/runner/junit5/JUnitTestRunnerListener.kt

```diff
@@ -109,9 +103,9 @@ class JUnitTestRunnerListener(private val listener: EngineExecutionListener,
   }
 
   private fun writeSpecFailures(failures: List<ResultState>): Try<Any> = Try {
-    val dir = Paths.get(".kotlintest")
-    dir.toFile().mkdirs()
-    val path = dir.resolve("spec_failures").toAbsolutePath()
+    val kotlintestBuildDirectory = Paths.get("build/.kotlintest")
```

Comment:
build isn't used by everyhing
maven uses `target`
some other things use `out`

Reply:
Hmm.. You're right. What can we do in this case then?

Reply:
I think we should have the gradle plugin provide it as a parameter to the test runner. That's the best way to do it, but we don't control the junit platform plugin. It might already supply it.